docs/local-gadget.md

```diff
@@ -289,6 +289,23 @@ CONTAINER        PID     COMM             IP  SADDR            DADDR
 test-container   503650  wget             4   172.17.0.3       93.184.216.34    80
 ```
 
+### Trace/Signal
+
+The signal trace gadget is used to trace system signals received by containers.
+
+```
+docker run --name nginx -p 80:80 -d nginx
+docker exec -it nginx sh -c 'sleep 3'
```

Comment:
Even if this test container triggers some signals, I suggest doing something more explicit. What about this:

```bash
$ docker run -it --rm --name test-container busybox /bin/sh
/ # sleep 100 &
/ # echo $!
9
/ # kill -kill $!
/ #
[1]+  Killed                     sleep 100
/ #
```

```bash
$ sudo local-gadget trace signal -c test-container -r docker
CONTAINER                                     PID        COMM             SIGNAL      TPID       RET
test-container                                243291     runc:[2:INIT]    SIGURG      243291     0
test-container                                243291     runc:[2:INIT]    SIGURG      1          0
test-container                                243291     sh               SIGKILL     243399     0
test-container                                243291     sh               SIGKILL     9          0
```

What do you think?

I opened https://github.com/inspektor-gadget/inspektor-gadget/issues/1056 to investigate the duplicated entries.

Reply:
Makes sense. Updated.